episode-6.msrt

```diff
@@ -254,7 +252,7 @@
 51
 00:03:43,310 --> 00:03:47,310
 [ru] Зелье Зомбификации... глупости какие!
-[en] A potion of Zombification... what nonsense!
+[en] A potion of Zombification... that's nothing!
```

Comment:
I think saying "what nonsense" is a bit too dismissive for Saffron to say. She knows about Zombiah's powers, but thinks she has a potion that is so much better than just bringing a bird back to life. This is a core competency of Zombiah, and Saffron is saying "Yeah, you do this all the time. Only the rubes in the audience will think that's prize-worthy".